internal/provider/provider.go

```diff
@@ -130,7 +130,7 @@ func (p *BloxOneProvider) Resources(_ context.Context) []func() resource.Resourc
 
 		dfp.NewDfpResource,
 
-		fw.NewSecurityPoliciesResource,
+		fw.NewSecurityPolicyResource,
```

Comment:
AcessCodes, NamedLists, NetworkLists, InternalDomainLists all should be singular

Reply:
Updated